### PR TITLE
Cleanup additional license files

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
License should be placed only inside file headers.

### CODE REVIEW

File: analyzers/CBDE/LICENSE.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2012-2020 SonarSource SA
+Copyright (C) 2012-2021 SonarSource SA
```
Comment (pavel-mikula-sonarsource) [inline analyzers/CBDE/LICENSE.txt:1]: Leftover from happy new year.

I'm not really sure what to do with this file.

File: analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <title>SonarAnalyzer for C#</title>
     <authors>SonarSource</authors>
     <owners>SonarSource</owners>
-    <license type="file">License.txt</license>
+    <license type="expression">LGPL-3.0-only</license>
```
Comment (pavel-mikula-sonarsource) [inline analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec:9]: I manually checked nuget.org UI and this translates into link https://licenses.nuget.org/LGPL-3.0-only

File: docs/code-of-conduct.md [hunk 3]
```diff
@@ -1,71 +1,71 @@
...
-faith may face temporary or permanent repercussions as determined by other
-members of the project's leadership.
-
-## Attribution
-
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
-
-[homepage]: https://www.contributor-covenant.org
+## Our Pledge
```
Comment (pavel-mikula-sonarsource) [inline docs/code-of-conduct.md:1]: Moved file to keep project root clean

File: README.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ languages in [SonarQube](http://www.sonarqube.org/), [SonarCloud](https://sonarc
 ## How to contribute
 
 There are many ways you can contribute to the `sonar-dotnet` project. 
-When contributing, please respect our [Code of Conduct](./CODE_OF_CONDUCT.md).
+When contributing, please respect our [Code of Conduct](./docs/code-of-conduct.md).
```
Comment (pavel-mikula-sonarsource) [inline README.md:53]: This is just cleanup of the root directory (moved file).

And the end of this file, there's `## License` section. That's probably fine (?).

File: analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec (hunk 2 above)
Comment (andrei-epure-sonarsource) [reply analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec:9]: nice!

File: README.md (hunk 4 above)
Comment (andrei-epure-sonarsource) [reply README.md:53]: I'd keep the license in the root, as it is

File: LICENSE.txt [hunk 5]
```diff
@@ -1,165 +0,0 @@
-                   GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (andrei-epure-sonarsource) [inline LICENSE.txt:1]: please revert this change, we still need the license in the root

File: NOTICE.txt [hunk 6]
```diff
@@ -1,6 +0,0 @@
-SonarQube C# Plugin
```
Comment (andrei-epure-sonarsource) [inline NOTICE.txt:1]: given we have this notice in all our repos, let's revert this until we clarify if it's needed.
I will ask around

**Review by andrei-epure-sonarsource — CHANGES_REQUESTED** [review]
Let's keep LICENSE and NOTICE in the root folder to be aligned with all our other open-source repositories.

File: README.md (hunk 4 above)
Comment (pavel-mikula-sonarsource) [reply README.md:53]: Does it imply that you want to remove the `## License` section from `readme.md`? As `license.txt` file is discussed below.

Comment (andrei-epure-sonarsource) [reply README.md:53]: I don't mind either way, TBH, we can leave it as is

**Review by andrei-epure-sonarsource — APPROVED** [review]
LGTM
